nservicebus/recoverability/configure-delayed-retries_config_core_[6.0,).partial.md

```diff
@@ -1,6 +1,6 @@
 ## Configuring Delayed Retries
 
- * `TimeIncrease`: A time span after which the time between retries increases. Default: 10 seconds (`00:00:10`).
+ * `TimeIncrease`: A time span by which the delay interval increases with each delayed delivery. Default: 10 seconds (`00:00:10`).
```

Comment:
> `TimeIncrease`: A time span by which the delay interval increases with each delayed delivery. Default: 10 seconds (`00:00:10`).

How about:

`TimeIncrease`: Specified as a TimeSpan. Specifies the delay interval for each retry attempt. This delay increases by the same timespan with each delayed delivery. For example, if the specified value is 10 seconds, i.e. `00:00:10`, then the first delayed retry will be at 10 seconds, the subsequent delayed retry will be 20 seconds and so on.